src/Integration.Vsix/Analysis/AnalysisConfigMonitor.cs

```diff
@@ -75,15 +71,8 @@ internal sealed class AnalysisConfigMonitor : IAnalysisConfigMonitor, IDisposabl
 
         private void OnUserSettingsChanged(object sender, EventArgs e)
         {
-            if (activeSolutionBoundTracker.CurrentConfiguration.Mode == SonarLintMode.Standalone)
-            {
-                logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsChanged);
-                OnSettingsChangedAsync().Forget();
-            }
-            else
-            {
-                logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsIgnoredForConnectedModeLanguages);
-            }
+            logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsChanged);
```

Comment:
I think it's worth adding a comment explaining why we might trigger re-analysis in Connected Mode e.g. 

> There is a corner-case where we want to raise the the event even in Connected Mode - see #3701.